packages/sdk/src/sdkBase.ts

```diff
@@ -806,7 +804,7 @@ export class NxtpSdkBase {
       return BigNumber.from(a.bid.amountReceived).gt(b.bid.amountReceived) ? -1 : 1;
     })[0];
 
-    return { ...chosen, metaTxRelayerFee };
+    return { ...chosen, totalFee, metaTxRelayerFee, routerFee };
```

Comment:
Shouldn't the *actual* total fee and router fee returned here come from the bid itself? Ideally, it should be the same as our internal estimate above. But we should definitely be returning what the bid is really offering, just in case it is different for some reason.

Reply:
In fact, it could easily be different if gas price or token price has changed slightly since we did the estimate.